### PR TITLE
KOGITO-2921 Bump to Quarkus 1.7.0.Final

### PR DESCRIPTION
**Pull request by evacchi**
- https://github.com/kiegroup/kogito-runtimes/pull/682
- https://github.com/kiegroup/kogito-apps/pull/375
- https://github.com/kiegroup/kogito-examples/pull/322

### CODE REVIEW

**Comment by evacchi** [comment]
jenkins please retest this

File: pom.xml [hunk 1]
```diff
@@ -117,7 +117,7 @@
     <version.io.cloudevents>1.3.0</version.io.cloudevents>
     <version.io.fabric8.kubernetes-client>4.8.0</version.io.fabric8.kubernetes-client>
     <version.io.prometheus>0.5.0</version.io.prometheus>
-    <version.io.quarkus>1.6.0.Final</version.io.quarkus>
+    <version.io.quarkus>1.7.0.Final</version.io.quarkus>
     <version.io.restassured>4.3.0</version.io.restassured>
     <version.io.restassured.springboot>3.3.0</version.io.restassured.springboot>    <!-- Manually override to 3.3.0 because Spring boot uses rest-assured-common@3.3.0, otherwise error: The type io.restassured.common.mapper.TypeRef cannot be resolved. It is indirectly referenced from required .class files -->
     <version.io.smallrye.reactive>2.1.1</version.io.smallrye.reactive>
```
Comment (radtriste) [inline pom.xml:123]: Keeping that one ?

Comment (cristianonicolai) [reply pom.xml:123]: Guess it needs to be removed to be consistent with 0.13.x

**Comment by cristianonicolai** [comment]
@evacchi needs rebase

**Pull request merged**